2024-02-19 19:13:35,963 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 19:13:35,963 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Configure stats pid to 19360
2024-02-19 19:13:35,963 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 19:13:35,963 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 19:13:35,964 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 19:13:35,964 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 19:13:35,964 INFO    MainThread:19360 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_191335-r1rn51cj\logs\debug.log
2024-02-19 19:13:35,965 INFO    MainThread:19360 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_191335-r1rn51cj\logs\debug-internal.log
2024-02-19 19:13:35,965 INFO    MainThread:19360 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002A84F1CA7D0>
2024-02-19 19:13:35,965 INFO    MainThread:19360 [wandb_init.py:init():566] calling init triggers
2024-02-19 19:13:35,965 INFO    MainThread:19360 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'n_trees': 100, 'min_samples': 5, 'max_depth': 2}
2024-02-19 19:13:35,965 INFO    MainThread:19360 [wandb_init.py:init():591] re-initializing run, found existing run on stack: jk0woqwk
2024-02-19 19:13:35,967 INFO    MainThread:19360 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/jk0woqwk
2024-02-19 19:13:35,967 INFO    MainThread:19360 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 19:13:35,967 INFO    MainThread:19360 [wandb_run.py:_restore():2193] restore
2024-02-19 19:13:35,968 INFO    MainThread:19360 [wandb_run.py:_restore():2199] restore done
2024-02-19 19:13:39,745 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 19:13:39,745 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 19:13:39,751 INFO    MainThread:19360 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-19 19:13:39,845 INFO    MainThread:19360 [wandb_init.py:init():616] starting backend
2024-02-19 19:13:39,845 INFO    MainThread:19360 [wandb_init.py:init():620] setting up manager
2024-02-19 19:13:39,847 INFO    MainThread:19360 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-19 19:13:39,848 INFO    MainThread:19360 [wandb_init.py:init():628] backend started and connected
2024-02-19 19:13:39,857 INFO    MainThread:19360 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-19 19:13:39,857 INFO    MainThread:19360 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-19 19:13:39,857 INFO    MainThread:19360 [wandb_init.py:init():720] updated telemetry
2024-02-19 19:13:39,858 INFO    MainThread:19360 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 19:13:41,363 INFO    MainThread:19360 [wandb_run.py:_on_init():2262] communicating current version
2024-02-19 19:13:41,688 INFO    MainThread:19360 [wandb_run.py:_on_init():2271] got version response 
2024-02-19 19:13:41,689 INFO    MainThread:19360 [wandb_init.py:init():804] starting run threads in backend
2024-02-19 19:13:41,776 INFO    MainThread:19360 [wandb_run.py:_console_start():2241] atexit reg
2024-02-19 19:13:41,776 INFO    MainThread:19360 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-19 19:13:41,776 INFO    MainThread:19360 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-19 19:13:41,776 INFO    MainThread:19360 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-19 19:13:41,777 INFO    MainThread:19360 [wandb_init.py:init():847] run started, returning control to user process
2024-02-19 19:29:35,194 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 19:29:35,195 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Configure stats pid to 19360
2024-02-19 19:29:35,195 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 19:29:35,196 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 19:29:35,196 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 19:29:35,196 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 19:29:35,197 INFO    MainThread:19360 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_192935-hox0lwhf\logs\debug.log
2024-02-19 19:29:35,197 INFO    MainThread:19360 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_192935-hox0lwhf\logs\debug-internal.log
2024-02-19 19:29:35,197 INFO    MainThread:19360 [wandb_init.py:init():566] calling init triggers
2024-02-19 19:29:35,197 INFO    MainThread:19360 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'n_trees': 100, 'min_samples': 5, 'max_depth': 10}
2024-02-19 19:29:35,198 INFO    MainThread:19360 [wandb_init.py:init():591] re-initializing run, found existing run on stack: r1rn51cj
2024-02-19 19:29:35,200 INFO    MainThread:19360 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/r1rn51cj
2024-02-19 19:29:35,200 INFO    MainThread:19360 [jupyter.py:save_history():445] not saving jupyter history
2024-02-19 19:29:35,201 INFO    MainThread:19360 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-19 19:29:35,201 INFO    MainThread:19360 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-19 19:29:35,201 INFO    MainThread:19360 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 19:29:35,202 INFO    MainThread:19360 [wandb_run.py:_restore():2193] restore
2024-02-19 19:29:35,202 INFO    MainThread:19360 [wandb_run.py:_restore():2199] restore done
2024-02-19 19:29:38,801 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 19:29:38,802 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 19:29:38,808 INFO    MainThread:19360 [wandb_run.py:_footer_sync_info():3825] logging synced files
